server/vcr-server/api/v4/serializers/rest/credential.py

```diff
@@ -55,6 +55,22 @@ class Meta:
             "schema",
         )
 
+class CredentialTypeClaimLabelsSerializer(ModelSerializer):
```

Comment:
Is `CredentialTypeSerializer` still being used anywhere? If not then we can just modify it rather than add a new serializer.

Reply:
yes it's being used in the rest serializer which is in turn being used in v4/views/rest/topic.py